### ISSUE TITLE
Pagination options setting max records

### ISSUE THREAD

**Issue by nWidart** [njleonzhang/vue-data-tables#89]
Hello,

I'm setting up server side fetching of paginated results.

From the documentation I see we can only set the following options:

```js
paginationDef: {
    pageSize: 1,
    pageSizes: [1, 2, 3],
    currentPage: 2
}
```

Is there a way to tell the package how many total records there are, and thus prepare the paginator element with the appropriate amount of pages?

Thanks,

**Comment by njleonzhang** [comment]
I am working on server side version now, will release ASAP

**Comment by njleonzhang** [comment]
released in 2.1.0
